godotproject/project.godot

```diff
@@ -179,6 +179,10 @@ menu={
 [rendering]
 
 quality/driver/driver_name="GLES2"
+quality/2d/use_pixel_snap=true
 vram_compression/import_etc=true
 vram_compression/import_etc2=false
+batching/precision/uv_contract=true
```

Comment:
The "Use pixel snap" and "UV contract" settings will be applied on all platforms, to ensure our pixel graphics are crisp.